scania-digital-design-system/tegel
# fix(message): updated storybook mode-variant default

**Describe pull-request**  
remove mode-variant showing as default but twice if changed to primary...

**Solving issue**  
Fixes: [CDEP-3028](https://tegel.atlassian.net/browse/CDEP-3028)

**How to test**  
1. Go to storybook and message component
2. Make sure default no mode-variant is showing but it should show if you change the prop from "Inherit from parent"

**Suggested test steps**


[CDEP-3028]: https://tegel.atlassian.net/browse/CDEP-3028?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
